hwilib/commands.py

```diff
@@ -245,6 +252,10 @@ def process_commands(args):
     if command == 'enumerate':
         return args.func(args)
 
+    if hasattr(args, "desc") and args.desc:
```

Comment:
I don't think this if-block is useful. Anything that has a `desc` option should be handling it by itself inside of it's own function.

Reply:
I don't like this either, but it was throwing errors if I didn't set fingerprint early enough.

Reply:
I replaced this with a consistency check if both `--desc` and `--fingerprint` is set.